newton/_src/sim/model.py

```diff
@@ -36,13 +36,13 @@ class Model:
 
     Key Features:
         - Stores all static data for simulation: particles, rigid bodies, joints, shapes, soft/rigid elements, etc.
-        - Supports grouping of entities by environment using group indices (e.g., `particle_group`, `body_group`, etc.).
-          - Group index -1: global entities shared across all environments.
-          - Group indices 0, 1, 2, ...: environment-specific entities.
+        - Supports grouping of entities by world using group indices (e.g., `particle_group`, `body_group`, etc.).
```

Comment:
Those should be called `particle_world`, `body_world`, etc.
Also refer to them as "world indices".